week1/02.py

```diff
@@ -0,0 +1,6 @@
+your_favorite_movies = []
+i = 0
+while i < 5 :
```

Comment:
You should only use while loops if you really need to, it's better practice to use for loops in python.
A for loop that counts from 0 to 4 looks like this:
```python
for i in range(5):
    print(i)
```